src/sounds.cpp

```diff
@@ -619,9 +620,9 @@ void sfx::generate_gun_sound( const player &p, const item &firing )
 
         const auto mods = firing.gunmods();
         if( std::any_of( mods.begin(), mods.end(),
-                         []( const item * e ) {
-                             return e->type->gunmod->loudness < 0;
-                         } ) ) {
+        []( const item * e ) {
+        return e->type->gunmod->loudness < 0;
+    } ) ) {
```

Comment:
I know astyle did this, but what?  It seems super confused here.

Reply:
What if you move the lambda up to `if` does that make it less confused?

`f( std::any_of( mods.begin(), mods.end(), []( const item * e ) {`

Or maybe make a variable for a lambda?

```
auto sel = []( const item * e) {
    return e->type->gunmod->loudness < 0;
};
if( std::any_of( mods.begin(), mods.end(), sel ) ) {
```